bin/pycbc_multi_inspiral

```diff
@@ -16,56 +16,56 @@
 # with this program; if not, write to the Free Software Foundation, Inc.,
 # 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-
 import logging
+import time
 from collections import defaultdict
 import argparse
-from pycbc import vetoes, psd, waveform, events, strain, scheme, fft,\
+import numpy as np
+from pycbc import (
+    detector, fft, inject, opt, psd, scheme, strain, vetoes, waveform,
     DYN_RANGE_FAC
-from pycbc.vetoes import sgchisq
+)
+from pycbc.events import coherent as coh, EventManagerCoherent
 from pycbc.filter import MatchedFilterControl
 from pycbc.types import TimeSeries, zeros, float32, complex64
 from pycbc.types import MultiDetOptionAction
-import pycbc.detector
-import numpy as np
-
-import time
+from pycbc.vetoes import sgchisq
 
 time_init = time.time()
 
-parser = argparse.ArgumentParser(usage='',
-    description="Find multiple detector gravitational-wave triggers.")
+parser = argparse.ArgumentParser(
+    usage="", description="Find multiple detector gravitational-wave triggers."
+)
```

Comment:
I suggest putting the description in the docstring at the top of the file, and then doing
```python
parser = argparse.ArgumentParser(description=__doc__)
```
See for example `pycbc_optimal_snr` or other scripts.